app/controllers/api/v1/books_controller.rb

```diff
@@ -1,15 +1,12 @@
 module Api
   module V1
-    class BooksController < ApplicationController
-      before_action :authenticate_user!
-
-      include Wor::Paginate
-
+    class BooksController < ApiController
       def index
         render_paginated Book
       end
 
       def show
+        # Not a strong parameter, because :id is part of the API URL
         @book = Book.find(params[:id])
```

Comment:
What about when no book is found? You should return a 404

Reply:
Book.find raises a RecordNotFound exception and, at least, in development it returns 404 with the message Couldn't find Book with 'id'=10, but for sanity, I'll implement a nicer JSON response.